src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/ReviewUnusedParametersTests.cs

```diff
@@ -965,7 +965,7 @@ Solution WithAnalyzerConfigDocument(Solution solution, ProjectId projectId)
                     .AddAnalyzerConfigDocument(
                         ".editorconfig",
                         SourceText.From($"[*.{documentExtension}]" + Environment.NewLine + editorConfigText),
-                        filePath: @"z:\.editorconfig")
+                         filePath: @"/tmp/.editorconfig")
```

Comment:
This will break Windows, I suspect.

We probably need to figure out if the platform is Windows or Linux (macOS uses similar paths) and then pass an appropriate path.

Reply:
This is what we use in Roslyn repo: https://github.com/dotnet/roslyn/blob/a8de4256f76ea0ce06bb7683747ee615c9272147/src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeRefactoringVerifier%601%2BTest.cs#L55

Maybe that works?

Reply:
This is a valid path on windows

![image](https://user-images.githubusercontent.com/9797472/99851455-0e1ce300-2b34-11eb-9376-0eae70f81ae7.png)

Reply:
@mavasani removing `tmp` from the path is fine. I assume the only thing we should change is making sure the paths are valid in both windows and unix.

Reply:
Cool, I am fine with anything that makes the test pass :-)